evm/upgradable.js

```diff
@@ -47,7 +47,7 @@ async function deployCreate2Upgradable(
     implementationConstructorArgs = [],
     proxyConstructorArgs = [],
     setupParams = '0x',
-    key = Date.now(),
+    salt = Date.now(),
```

Comment:
should be a mandatory arg to force user to decide

Reply:
removed default value